examples/Cargo.toml

```diff
@@ -32,6 +36,10 @@ femtovg-support = ["epoxy", "femtovg", "glow", "libloading"]
 glium-support = ["glium", "epoxy", "libloading"]
 v4_12 = ["gtk/v4_12"]
 v4_14 = ["gtk/v4_14"]
+tokio-support = ["tokio"]
+async-channel-support = ["async-channel"]
+reqwest-support = ["reqwest"]
+serde-support = ["serde"]
```

Comment:
For now, those features are only used by one example. So I would group them into a single feature and just call it `tokio = ["dep:tokio", "async-channel", "reqwest", "serde"]`